src/views/TutorialsOverview.vue

```diff
@@ -78,6 +90,7 @@ export default {
       this.$nextTick(() => {
         // Send a 'rendered' message to the host when new data has been patched onto the DOM.
         this.newContentMounted();
+        this.store.setReferences(this.overviewProps.references);
```

Comment:
If you really want to avoid adding this code here, you may be able to move the setting of the references into the related `/component` child of this page, similar to how you do it for the Article and Topic. 

The `references` at that point will be re-calculated as the whole component is re-rendered.